warehouse/packaging/tasks.py

```diff
@@ -56,6 +57,35 @@ def sync_file_to_cache(request, file_id):
         file.cached = True
 
 
+@tasks.task(ignore_result=True, acks_late=True)
+def compute_packaging_metrics(request):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏what schedules this task?

Reply:
https://github.com/pypi/warehouse/pull/14417